projects/worker-azure-ai/actions/worker-azure-tagging/index.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+const { worker, GenericError } = require("@adobe/asset-compute-sdk");
+const fetch = require("node-fetch");
+const fs = require('fs').promises;
+
+const DEFAULT_LANGUAGE = "en";
+
+exports.main = worker(async (source, rendition, params) => {
+    // get Azure credentials for API
+    let subscriptionKey = params && params.AZURE_OCP_KEY;
+    let endpoint = params && params.AZURE_OCP_ENDPOINT;
+
+    // WORKER_TEST_MODE is true when you are running worker unit
+    // tests using the `test-worker` command
+    // Dummy authorization parameters needed for mocks 
+    if (process.env.WORKER_TEST_MODE) {
+        subscriptionKey = "test-azure-key";
+        endpoint = "https://westus.api.cognitive.microsoft.com/";
+    }
```

Comment:
I would suggest `if/else` here with below condition. I nearly missed the below condition and asked to check it after assignments! ^^'